src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Marshaling/JSMarshalerArgument.Func.cs

```diff
@@ -139,6 +140,9 @@ public unsafe void ToManaged(out Action? value)
         /// Implementation of the argument marshaling.
         /// It's used by JSImport code generator and should not be used by developers in source code.
         /// </summary>
+        /// <typeparam name="T">Type of marshaled argument of the Action</typeparam>
```

Comment:
Suggestion applies to all similar texts.
Since MS Docs already shows the full type of the `Action<T1, T2>` argument in the signature, we can lowercase it here:

```suggestion
        /// <typeparam name="T">The type of the marshaled argument of the action.</typeparam>
```